build.gradle.kts

```diff
@@ -110,6 +110,8 @@ dependencies {
     testImplementation("org.jenkins-ci.plugins:pipeline-stage-step:2.3")
     testImplementation("org.jenkins-ci.plugins:pipeline-maven:3.10.0")
     testImplementation("org.spockframework:spock-core:1.3-groovy-2.4")
+    testImplementation("net.bytebuddy:byte-buddy:1.12.20")
+    testImplementation("org.objenesis:objenesis:3.3")
```

Comment:
Why are these needed? I couldn't find them being used in tests.

Reply:
It is needed to be able to mock classes and classes without the default constructor in Spock.

Reply:
They are used by the framework under the hood.